src/modules/75-cf/pages/feature-flags-detail/targeting-rules-tab/hooks/useFeatureEnabled.tsx

```diff
@@ -16,6 +16,8 @@ interface UseFeatureEnabledReturn {
   enabledByPermission: boolean
   enabledByPlanEnforcement: boolean
   featureEnabled: boolean
+  canEdit: boolean
+  canToggle: boolean
 }
 
 const useFeatureEnabled = (): UseFeatureEnabledReturn => {
```

Comment:
(cont from above) Permission gets passed from 

```suggestion
const useFeatureEnabled = (permission): UseFeatureEnabledReturn => {
```